CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ CHANGELOG
 
 _**Note:** This is in reverse chronological order, so newer entries are added to the top._
 
+Swift Next
+----------
+* [SE-0290][]:
+
+  It is now possible to write inverted availability statements by using the new `#unavailable` keyword:
```

Comment:
cc @amartini51 @beccadax is "keyword" the correct terminology here or should it be something else?

For example, https://forums.swift.org/t/concise-magic-file-names/31297 uses "magic identifier" to describe #file.

Reply:
`availability statements` → `availability conditions`
https://docs.swift.org/swift-book/ReferenceManual/Statements.html#ID522
https://docs.swift.org/swift-book/LanguageGuide/ControlFlow.html#ID523

Reply:
Calling `#unavailable` a keyword matches `#available`, which we include in the list of keywords here:

https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID413